src/Atmos/Model/AtmosModel.jl

```diff
@@ -205,19 +246,33 @@ struct AtmosModel{FT, PH, PR, O, S, DC} <: BalanceLaw
     data_config::DC
 end
 
-parameter_set(atmos::AtmosModel) = atmos.physics.param_set
-moisture_model(atmos::AtmosModel) = atmos.physics.moisture
-energy_model(atmos::AtmosModel) = atmos.physics.energy
-compressibility_model(atmos::AtmosModel) = atmos.physics.compressibility
-reference_state(atmos::AtmosModel) = atmos.physics.ref_state
-turbulence_model(atmos::AtmosModel) = atmos.physics.turbulence
-turbconv_model(atmos::AtmosModel) = atmos.physics.turbconv
-hyperdiffusion_model(atmos::AtmosModel) = atmos.physics.hyperdiffusion
-viscoussponge_model(atmos::AtmosModel) = atmos.physics.viscoussponge
-precipitation_model(atmos::AtmosModel) = atmos.physics.precipitation
-radiation_model(atmos::AtmosModel) = atmos.physics.radiation
-tracer_model(atmos::AtmosModel) = atmos.physics.tracers
-lsforcing_model(atmos::AtmosModel) = atmos.physics.lsforcing
+parameter_set(atmos::AtmosModel) = parameter_set(atmos.physics)
+moisture_model(atmos::AtmosModel) = moisture_model(atmos.physics)
+energy_model(atmos::AtmosModel) = energy_model(atmos.physics)
+compressibility_model(atmos::AtmosModel) = compressibility_model(atmos.physics)
+reference_state(atmos::AtmosModel) = reference_state(atmos.physics)
+turbulence_model(atmos::AtmosModel) = turbulence_model(atmos.physics)
+turbconv_model(atmos::AtmosModel) = turbconv_model(atmos.physics)
+hyperdiffusion_model(atmos::AtmosModel) = hyperdiffusion_model(atmos.physics)
+viscoussponge_model(atmos::AtmosModel) = viscoussponge_model(atmos.physics)
+precipitation_model(atmos::AtmosModel) = precipitation_model(atmos.physics)
+radiation_model(atmos::AtmosModel) = radiation_model(atmos.physics)
+tracer_model(atmos::AtmosModel) = tracer_model(atmos.physics)
+lsforcing_model(atmos::AtmosModel) = lsforcing_model(atmos.physics)
+
+parameter_set(physics::AtmosPhysics) = physics.param_set
```

Comment:
@kpamnany what do you think about defining the same method for `AtmosPhysics`? Does this look okay?

Reply:
Looks fine to me.